en/docs/learn/examples/rest-api-examples/configuring-non-http-endpoints.md

```diff
@@ -7,20 +7,20 @@ Following is a sample REST API configuration that we can used to implement this
 
```

Comment:
Following is a sample REST API configuration that we can used to implement this scenario. 
->

Following is a sample REST API configuration that we can use to implement this scenario.